test/extended/testdata/egress-firewall/ovnk-egressfirewall-wildcard-test.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: default
+spec:
+  egress:
+  - type: Allow
+    to:
+      dnsName: docs.openshift.com
+  - type: Allow
+    to:
+      dnsName: "*.ci.openshift.org"
```

```suggestion
  - type: Allow
    to:
      dnsName: "*.google.com"
```
We can utilize the exiting curl to `www.google.com` for the wildcard DNS rule.